src/style.css

```diff
@@ -0,0 +1,131 @@
+html {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+}
+
+.card {
+  display: inline-block;
+  font-size: 16px;
+}
+
+.imac_photo {
+  width: 160px;
+  height: 134px;
+  padding: 32px 19px 36px;
+}
+
+.container__card,
+.card {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px 16px 24px;
+}
+
+.product_name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  padding: 0 16px;
+  color: #060b35;
+  margin: 0;
```

Comment:
check order of css properties 
https://codeguide.academy/html-css.html#css-order